homework-submission/index.js

```diff
@@ -0,0 +1,75 @@
+/*to create a list item <li> element whose props are description and deadline*/
+
+const ListItem = ({ description, deadline, className }) => {
+  return (
+    <li className={className}>
+      Description: {description} Deadline: {deadline}
+    </li>
+  );
+};
+
+const StaticList = () => {
+  return (
+    <ul>
+      <p>This is the Static List</p>
```

Comment:
Paragraph in a `ul` tag may not be a good choice, If I were you I would probably think about that again.

Let's say you have a program with lots of lists and items, and it's an ordered list (or an un-ordered):

 - javascript will look to the `ul` tag to see what to do  with it, and it figures out it suppose to add numbers or pullets to each element as convention of a list
 - then javascript will look to each item on it's own to recognize either it's a list item or it's a sub list
 - if a list item found it will put the number/pullet otherwise it will start another list with other kind of a (pullet/number) style

for anything other than these it won't be able to figure out what is happening with it's list items and sub-lists, and in some programs it **_may_** ignore the whole convention it follows.
Css may act the same as that, especially if you are in a team with lots of people looking to each line and trying to figure out why is this line isn't acting as they asked for?

for more details about how you can know which you can put where, please look the documentation:
 - at the [Permitted content](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ul) part below the example given.